composer.json

```diff
@@ -34,7 +34,8 @@
     "behat/mink": "~1.5",
     "behat/mink-extension": "~2.0",
     "behat/mink-browserkit-driver": "~1.1",
-    "behatch/contexts": "~2.1"
+    "behatch/contexts": "dev-master#eef7ab39ca896796bf3ba25a0fa5a95f15276eb7",
```

Comment:
The Given step for creating dummies fail in some occasion since the removal of the 2.2 tag when running features individually.